xbmc/addons/AddonCallbacksPVR.cpp

```diff
@@ -191,8 +191,7 @@ void CAddonCallbacksPVR::PVRTransferRecordingEntry(void *addonData, const ADDON_
   }
 
   /* transfer this entry to the recordings container */
-  CPVRRecording transferRecording(*recording, client->GetID());
-  xbmcRecordings->UpdateFromClient(transferRecording);
+  xbmcRecordings->UpdateFromClient(*recording, client->GetID());
```

Comment:
no, add-on C types are not to be passed around. we translate to a CPVRRecording in here.